Sources/PointFree/Blog/BlogPosts/AllBlogPosts.swift

```diff
@@ -18,4 +18,5 @@ private let _allBlogPosts: [BlogPost] = [
   post0007_openSourcingNonEmpty,
   post0008_conditionalCoding,
   post0009_6moAnniversary,
+  post0010_studentDiscounts,
```

Comment:
could use some proof reading on the blog post. it's kinda slim so if you have some ideas of how to beef it up I'm all ears.